sidecar/proxy/envoy/resources.go

```diff
@@ -70,6 +70,7 @@ type Route struct {
 	PrefixRewrite string   `json:"prefix_rewrite,omitempty"`
 	Cluster       string   `json:"cluster"`
 	Headers       []Header `json:"headers,omitempty"`
+	Timeout       string   `json:"timeout_ms,omitempty"`
```

Comment:
Was this added accidentally? I don't see it used anywhere, and it is type `string`.